docs/metrics.md

```diff
@@ -90,8 +90,8 @@ Extra labels:
 * `drive` - Disk device that is being written/read.
 * `type` - Whether it's a read or write operation.
 
-#### kubevirt_vmi_storage_times_ms_total
-#### HELP kubevirt_vmi_storage_times_ms_total storage operation time.
+#### kubevirt_vmi_storage_times_ns_total
```

Comment:
oh wait, that should be `kubevirt_vmi_storage_times_seconds_total`.

Reply:
Yep, that's what I was talking about `make generate` not working as expected...

It is not changing VMI metrics, only adding them at the end of the file as "Other metrics".
I'll just change manually for now

Reply:
Should be go to go now

Reply:
Oh, looks like pull_kubevirt_generate fails if I change ir manually :(

Reply:
Should be good to go now